eras/shelley-ma/test-suite/cardano-ledger-shelley-ma-test.cabal

```diff
@@ -34,12 +34,12 @@ library
     hs-source-dirs:   src
     default-language: Haskell2010
     ghc-options:
-        -Wall -Wcompat -Werror -Wincomplete-record-updates
+        -Wall -Wcompat -Wincomplete-record-updates
         -Wincomplete-uni-patterns -Wpartial-fields -Wredundant-constraints
         -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <4.21,
```

Comment:
I suggest switching all packages to `<5` on `base`.

Reply:
Done!